files/sysbus/org.webosports.luna-qml-launcher.perm.json

```diff
@@ -0,0 +1,14 @@
+{
+    "org.webosports.*": [
+        "all"
+    ],
+    "com.palm.*": [
+        "all"
+    ],
+    "": [
```

Comment:
ideally, "" and "*" will disappear when we learn how to correctly make it behave as a "container" or "launcher"

Reply:
Yes agree, but this should work for now at least. Can always restrict it more down the line.